### ISSUE TITLE
Error Messages 

### ISSUE THREAD

**Issue by riteshpushkar** [algovengers/Hairify#146]
(https://github.com/algovengers/Hairify/assets/79689579/a31fa377-86a4-49c3-bc64-19f506a0adeb)
As you can see in this image that error message is not looking good moreover not even visible at once I would like to add some css to it and make it attractive
Pls assign me this under gssoc24

**Comment by afeefuddin** [comment]
Assigned @riteshpushkar

**Comment by afeefuddin** [comment]
Are you planning to complete this @riteshpushkar?

**Comment by riteshpushkar** [comment]
Yes @afeefuddin I'll be completing it asap. Thank you for your patience
